src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,58 @@
+import { Component } from 'react';
+
+type Props = {
+  clockName: string,
+};
+
+type State = {
+  today: Date,
+};
+
+const formatDate = (value: Date) => value.toUTCString().slice(-12, -4);
+
+export class Clock extends Component<Props, State> {
+  state: Readonly<State> = {
+    today: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      this.setState({ today: new Date() });
+
+      window.console.info(formatDate(this.state.today));
```

Comment:
Don't use state here, it can be outdated
Save new Date() result to constant and reuse it for both setState and console.info